insights/specs/insights_archive.py

```diff
@@ -83,6 +83,7 @@ class InsightsArchiveSpecs(Specs):
     gluster_peer_status = simple_file("insights_commands/gluster_peer_status")
     hammer_ping = simple_file("insights_commands/hammer_ping")
     hammer_task_list = simple_file("insights_commands/hammer_--config_.root..hammer.cli.modules.d.foreman.yml_--output_csv_task_list_--search_state_running_AND_label_Actions_Candlepin_ListenOnCandlepinEvents_OR_label_Actions_Katello_EventQueue_Monitor")
+    hammer_compute_resource_list = simple_file("hammer --interactive 0 --output json compute-resource list")
```

Comment:
The string to the `simple_file` is what it would be in the insights_archives. it can be generated by the `mangle` command of insights-core:

```
# cd insights-core
# source [path_to_venv]/bin/activate
# mangle "/usr/bin/hammer --interactive 0 --output json compute-resource list"
"insights_commands/hammer_--interactive_0_--output_json_compute-resource_list"
```